uvicorn/workers.py

```diff
@@ -72,6 +72,11 @@ def init_signals(self) -> None:
         for s in self.SIGNALS:
             signal.signal(s, signal.SIG_DFL)
 
+        # Remove me if SIGUSR1 is handled in server.py
```

Comment:
I didn't get this comment here... This doesn't affect `uvicorn` standalone. 🤔

Reply:
I was referring to the comment from tiangolo.

> This is because the UvicornWorker, which inherits from the base Gunicorn worker, declares a method init_signals() (overriding the parent method) but doesn't do anything. I suspect it's because the signal handlers are declared in the Server.install_signal_handlers() with compatibility with asyncio.

But you are right, I was thinking that install_signal_handlers() in server.py *could* call init_signal in future. But this is unlikely as init_signal (esp signal.signal) isn't compatible with asyncio. I will remove this comment.